pyproject.toml

```diff
@@ -1,13 +1,16 @@
 [build-system]
-requires = ["poetry-core>=1.1.0a6"]
+requires = ["poetry-core>=1.2.0b2"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.mypy]
+python_version = 3.9
```

Comment:
Should this be the minimum we support?

Reply:
Will change to 3.8 to align with the python minimum dependency in [tool.poetry.dependencies].